general/payment_acquirers/payment_acquirers.rst

```diff
@@ -0,0 +1,180 @@
+      Directly. :doc:`Click here <../../accounting/receivables/customer_payments/batch_sdd>` for
+      more information about this payment method.
+
+- Online Payment Providers
+
+  - Adyen
+  - AliPay
+  - :doc:`Authorize.Net <authorize>`
+  - Buckaroo
+  - Ingenico
+  - :doc:`Paypal <paypal>`
+  - PayU
+  - PayUmoney
+  - Sips
+  - Stripe
+
+.. note::
+   Some of these Online Payment Providers can also be added as :doc:`Bank Accounts
+   <../../accounting/bank/setup/bank_accounts>`, but this is **not** the same process as adding them
+   as Payment Acquirers. Payment Acquirers allow customers to pay online, and Bank Accounts are
+   added and configured on your Accounting app to do a bank reconciliation, which is an accounting
+   control process.
+
+.. _payment_acquirers/configuration:
+
+Configuration
+=============
+
+Add a new Payment Acquirer
+--------------------------
+
+To add a new Payment acquirer and make it available to your customers, go to
+:menuselection:`Accounting --> Configuration --> Payment Acquirers`, look for your payment acquirer,
+install the related module, and activate it. To do so, open the payment acquirer and change its
+state from *Disabled* to *Enabled*.
+
+.. image:: media/payment-acquirers-activation.png
+   :align: center
+   :alt: Click on install, then on activate to make the payment acquirer available on Odoo.
+
+.. warning::
+   We recommend using the *Test Mode* on a duplicated database or a test database. The Test Mode is
+   meant to be used with your test/sandbox credentials, but Odoo generates Sales Orders and Invoices
+   as usual. It isn't always possible to cancel an invoice, and this could create some issues with
+   your invoices numbering if you were to test your payment acquirers on your main database.
+
+Credentials tab
+~~~~~~~~~~~~~~~
+
+If not done yet, go to the **Online Payment Provider**'s website, create an account, and make sure
+to have the credentials required for third-party use. Odoo requires these credentials to communicate
+with the Payment Acquirer and get the confirmation of the *payment authentication*.
+
+The form in this section is specific to the Payment Acquirer you are configuring. Please refer to
+the related documentation for more information.
+
+Configuration tab
+~~~~~~~~~~~~~~~~~
+
+You can change the Payment Acquirer front-end appearance by modifying its name under the **Displayed
+as** field and which credit card icons to display under the **Supported Payment Icons** field.
+
+Save and reuse Credit Cards
+***************************
+
+With the **Save Cards** feature, Odoo can store **Payment Tokens** in your database, which can be
+used for subsequent payments, without having to reenter the payment details. This is particularly
+useful for subscriptions' recurring payments.
+
+.. note::
+   The payment tokens stored on the database don't hold information about the credit cards used.
```

Comment:
```suggestion
   No credit card number gets directly saved in database; instead, payment tokens store a unique reference to the card which is saved on the payment acquirer's servers. This reference is useless without your credentials for the acquirer.
```
we do hold information (e.g. last 4 digits)

Reply:
I modified the note in the intro of the doc and removed it from this section. It was redundant.

> Odoo does not keep Credit Card numbers or credentials on its servers, nor is it stored on Odoo databases hosted elsewhere. Instead, Odoo apps use a unique reference to the data stored in the payment acquirers’ systems, where the information is safely stored. This reference is useless without your credentials for the payment acquirer.